src/pages/components/table.mdx

```diff
@@ -0,0 +1,210 @@
+  - tables/src/elements/HeaderRow.tsx
+  - tables/src/elements/OverflowButton.tsx
+  - tables/src/elements/Row.tsx
+  - tables/src/elements/SortableCell.tsx
+  - tables/src/elements/Table.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To organize and display data with logical relationships</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To structure content on a page, use the [Grid](/components/grid) component instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Table component.
+
+import Default from '../../examples/table/TableDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/table/TableDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Grouped rows
+
+Grouped rows are for additional categorization.
+
+import GroupedRows from '../../examples/table/TableGroupedRows.tsx';
+import GroupedRowsCode from '!!raw-loader!../../examples/table/TableGroupedRows.tsx';
+
+<CodeExample code={GroupedRowsCode}>
+  <GroupedRows />
+</CodeExample>
+
+### Overflow
+
+Overflow menus leverage the Dropdown and Menu component.
+
+import Overflow from '../../examples/table/TableOverflow.tsx';
+import OverflowCode from '!!raw-loader!../../examples/table/TableOverflow.tsx';
+
+<CodeExample code={OverflowCode}>
+  <Overflow />
+</CodeExample>
+
+### Pagination
+
+Table pagination works in combination with the [Pagination](/components/pagination) component.
+
+import Pagination from '../../examples/table/TablePagination.tsx';
+import PaginationCode from '!!raw-loader!../../examples/table/TablePagination.tsx';
+
+<CodeExample code={PaginationCode}>
+  <Pagination />
+</CodeExample>
+
+### Scroll
+
+To ensure that your table is read correctly by assistive technologies follow the
+[W3C Grid accessibility
+pattern](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html).
+
+import Scroll from '../../examples/table/TableScroll.tsx';
+import ScrollCode from '!!raw-loader!../../examples/table/TableScroll.tsx';
+
+<CodeExample code={ScrollCode}>
+  <Scroll />
+</CodeExample>
+
+### Selection
+
+Individual rows within a table can be selected for bulk actions.
+
+import SelectRows from '../../examples/table/TableSelectRows.tsx';
+import SelectRowsCode from '!!raw-loader!../../examples/table/TableSelectRows.tsx';
+
+<CodeExample code={SelectRowsCode}>
+  <SelectRows />
+</CodeExample>
+
+### Size
+
+Table row sizes comes in small, medium, and large. The default size is medium.
+
+import Size from '../../examples/table/TableSize.tsx';
+import SizeCode from '!!raw-loader!../../examples/table/TableSize.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Sort
+
+Columns can be sorted by selecting table headers.
+
+import Sort from '../../examples/table/TableSort.tsx';
+import SortCode from '!!raw-loader!../../examples/table/TableSort.tsx';
```

Comment:
There is some weirdness on the sorting controls. Recorded a screencast (with voice over) to demo the issue. https://share.getcloudapp.com/NQu1b1ll

I checked Styleguidist, and it looks like the focus thing is just how it works, but the second half of the video about the 2nd column's sorting functionality stands.

Reply:
The focus feels like a component bug. The action should match what we see under "Row Modifications" in this old CSS demo: https://zendeskgarden.github.io/css-components/tables/

Reply:
I filed a ticket for this component bug in Jira. I'll be sure the sorting functionality is working correctly (I believe JZ addressed it in a comment in this thread).